lib/exec.ml

```diff
@@ -187,6 +136,20 @@ let opam_switch_create_empty ~root () =
   let cmd = opam_cmd ~root "switch" % "create" % "empty" % "--empty" % "--no-install" in
   run_and_log cmd
 
+let run_opam_show ~root ~fields ~packages =
+  let fields = List.map (fun s -> s^":") fields in
```

Comment:
You should not concatenate `":"` here. The colon is part of the name of field and some fields do not have it.

Reply:
I agree, so now `:` is not treated specially, in order to avoid surprises

Reply:
You still pass field names without colon to `parse_string` but that's just for error messages.